### DISCUSSION TITLE
How to remove volumes for custom NEXTCLOUD_DATADIR path?

### DISCUSSION THREAD

**Discussion by zhuoqun-chen** [nextcloud/all-in-one#2257] [Questions] [answered]
Hello Developers,

I came across a problem: I set my own `NEXTCLOUD_DATADIR` to

```bash
-e NEXTCLOUD_DATADIR="/data"
```
And now I want to reset the failed AIO instance, and the `How to properly reset the instance?` section points out that we need to manully clean up the volumes if `NEXTCLOUD_DATADIR` has been set.

Could you please give me some help on how to clean the volumes up since I'm not familiar with docker? Thank you!

**Comment by Zoey2936** [comment, marked as answer]
I would say, just delete this folder

**Reply by zhuoqun-chen** [reply]
Thank you for the advice!
